scripts/bootstrap_bench_dual_source.py

```
@@ -0,0 +1,177 @@
+        print(f"login failed HTTP {status}: {body}", file=sys.stderr)
+        return ""
+    return str(body.get("token") or "")
+
+
+def bootstrap_direct(*, skip_bacnet_discover: bool) -> dict:
+    """Configure BACnet bench via existing scripts + model import."""
+    results: dict = {"bacnet": {}, "model": {}}
+    if not skip_bacnet_discover:
+        script = REPO / "scripts" / "apply_bench_four_points.sh"
+        if script.is_file():
+            subprocess.run([str(script), "--import-model"], check=False, cwd=REPO)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)